### PR TITLE
Code cleaning and editing

### PR DESCRIPTION
**Pull request by abdoachhoubi**
I replaced a legacy code in `client/src/index.js` from:
```js
ReactDOM.render(
<React.StrictMode>
    <BrowserRouter>
      <App />
    </BrowserRouter>
  </React.StrictMode>,
document.getElementById("root")
);
```
to:
```js
const root = ReactDOM.createRoot(document.getElementById("root"));
root.render(
<React.StrictMode>
    <BrowserRouter>
      <App />
    </BrowserRouter>
  </React.StrictMode>
);
```

I also added an export file in the `client/src/component`

I added some info in `manifest.json` and cleaned up `index.html`

I still need more information about the project to create a nice `README.md` file

### CODE REVIEW

**Comment by Pericles001** [comment]
> I replaced a legacy code in `client/src/index.js` from:
> 
> ```js
> ReactDOM.render(
> <React.StrictMode>
>     <BrowserRouter>
>       <App />
>     </BrowserRouter>
>   </React.StrictMode>,
> document.getElementById("root")
> );
> ```
> 
> to:
> 
> ```js
> const root = ReactDOM.createRoot(document.getElementById("root"));
> root.render(
> <React.StrictMode>
>     <BrowserRouter>
>       <App />
>     </BrowserRouter>
>   </React.StrictMode>
> );
> ```
> 
> I also added an export file in the `client/src/component`
> 
> I added some info in `manifest.json` and cleaned up `index.html`
> 
>Have you tested the client site after your updates and it was working ?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
-# devflix-video-streaming-app
-DevFlix is a video streaming app built in ReactJS and NodeJS &amp; Express as a part of the Software Engineering program by ALX AFRICA &amp; Holberton School.
+# Devflix
+
```
Comment (Pericles001) [inline README.md:2]: You can write some brief introduction to the project there , like: 
"DevFlix is a video streaming app built in ReactJS and NodeJS & Express as a part of the Software Engineering program by ALX AFRICA  & Holberton School."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+## Description
+
+## Key Features
+
+## Technologies
+
```
Comment (Pericles001) [inline README.md:10]: Client side: javascript, react
Server side: javascript, nodejs, express

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+## How to use
+
+## How to contribute
+
+## Contributors
```
Comment (Pericles001) [inline README.md]: Replace contributors by authors

File: README.md [hunk 4]
```diff
@@ -1,2 +1,15 @@
...
+
+## Acknowledgments
```
Comment (Pericles001) [inline README.md:30]: [Linode](https://linode.com) for their tutorial on [building react video streamin app](https://www.linode.com/docs/guides/build-react-video-streaming-app/#setting-up-the-server)

File: client/public/index.html [hunk 5]
```diff
@@ -1,45 +1,28 @@
...
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,200" rel="stylesheet" />
-  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css" rel="stylesheet" />
-  <title>React App</title>
-</head>
-
-<body>
-  <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (Pericles001) [inline client/public/index.html:31]: This noscript tag is needed , you may maintain it in the updated index.html

File: client/src/App.js [hunk 6]
```diff
@@ -1,41 +1,36 @@
 import React, { useState, useEffect } from 'react';
 
-import Login from './components/LoginPage';
-import Register from './components/RegisterPage';
-import Home from './components/HomePage';
-import TempPlayer from './components/TempPlayer';
-import firebase  from './services/firebase';
+import { Login, Register, Home, TempPlayer } from './components';
```
Comment (Pericles001) [inline client/src/App.js:3]: Is the new import statement working without any warnings or errors ? (line 3)

File: client/src/index.js [hunk 7]
```diff
@@ -1,20 +1,18 @@
...
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import ReactDOM from 'react-dom/client';
 import reportWebVitals from './reportWebVitals';
 import { BrowserRouter } from 'react-router-dom';
+import App from './App';
+import './index.css';
+
+const root = ReactDOM.createRoot( document.getElementById( "root" ) );
```
Comment (Pericles001) [inline client/src/index.js:8]: What are the advantages of the new codebase ?

File: README.md (hunk 2 above)
Comment (abdoachhoubi) [reply README.md:10]: Done

File: client/public/index.html [hunk 8]
```diff
@@ -1,45 +1,28 @@
...
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-</body>
-
-</html>
\ No newline at end of file
+  <body>
+    <div id="root"></div>
```
Comment (Pericles001) [inline client/public/index.html:27]:
```suggestion
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,30 @@
...
+
+  - JavaScript
+  - Node.js
+  - Express
+
+## How to use
+
+## How to contribute
+
+## Contributors
```
Comment (Pericles001) [inline README.md]:
```suggestion
## Authors
```

**Review by Pericles001 — CHANGES_REQUESTED** [review]
You did well but add the requested changes for approval

**Comment by abdoachhoubi** [comment]
> You did well but add the requested changes for approval

Sure!

File: client/src/App.js (hunk 6 above)
Comment (abdoachhoubi) [reply client/src/App.js:3]: It works well!

**Comment by Pericles001** [comment]
Merging this pull request should resolve issue #4

File: client/src/index.js (hunk 7 above)
Comment (abdoachhoubi) [reply client/src/index.js:8]: ReactDOM.render is no longer supported in React 18!
For more information read this [Deprecation Notice](https://stackoverflow.com/questions/71668256/deprecation-notice-reactdom-render-is-no-longer-supported-in-react-18)

File: client/src/App.js (hunk 6 above)
Comment (Pericles001) [reply client/src/App.js:3]: Great!

File: client/src/index.js (hunk 7 above)
Comment (Pericles001) [reply client/src/index.js:8]: Sure , you are right .

**Comment by abdoachhoubi** [comment]
> Merging this pull request should resolve issue #4

@Pericles001 Maybe but we still need to add more info in the README file.
I'll keep on adding information if I get more details about the project.

**Comment by Pericles001** [comment]
> > Merging this pull request should resolve issue #4
> 
> @Pericles001 Maybe but we still need to add more info in the README file. I'll keep on adding information if I get more details about the project.



> > Merging this pull request should resolve issue #4
> 
> @Pericles001 Maybe but we still need to add more info in the README file. I'll keep on adding information if I get more details about the project.

That is true. You will be assigned for further updates ...

**Review by Pericles001 — APPROVED** [review]

**Comment by abdoachhoubi** [comment]
I'll be happy to contribute anytime! ✌️

**Pull request merged**